native/src/components/SlideButton.tsx

```diff
@@ -7,12 +7,11 @@ type SlideButtonProps = {
   label: string
   onPress: () => void | Promise<void>
   theme: ThemeType
-  highlighted?: boolean
 }
 
-const SlideButton = ({ label, onPress, theme, highlighted }: SlideButtonProps): ReactElement => (
+const SlideButton = ({ label, onPress, theme }: SlideButtonProps): ReactElement => (
```

Comment:
i think there is no need for a separate component when just some styles change. This just should be another `variant` of a `<TextButton/>`